packages/create-swarmion-app/src/helpers/examples.ts

```diff
@@ -28,6 +28,14 @@ const isUrlOk = async (url: string): Promise<boolean> => {
   }
 };
 
+const getFileNameFromBranch = (branch: string): string => {
+  if (branch.match(/v(\d).(\d).(\d)/)) {
```

Comment:
You may encounted issues here with versions that have more than two digits. For example v1.12.488 is a valid version number but will not be matched

Reply:
Also we may want to have alpha or beta releases, how will this be handled ?

Reply:
Very good point

Reply:
I added tests to cover these use-cases 😄